setup.py

```diff
@@ -103,7 +103,12 @@ def read(filename):
         return f.read().decode('utf-8')
 
 
-dependencies = read('requirements.txt').split()
+dependencies = [
+    'click==6.7',
+    'psutil>=5.2.2,<6.0.0',
```

Comment:
I ran the tests with `psutil==5.4.0` and they worked. But I can change this line to whatever you'd like.